scripts/microshift-cleanup-data.sh

```diff
@@ -46,6 +46,10 @@ function stop_clean_pods() {
     # It is necessary to remove the pods (OVN-related last) to allow for further termination
     # of processes (i.e. conmon, etc.) that use the files under /var/run/ovn.
     # The cleanup of OVN data only works if the files under /var/run/ovn are not in use.
+    if [ ! -e /var/run/crio/crio.sock ]; then
+        echo "crio.sock is not present, not attempting to clean up pods"
+        return 0
```

Comment:
I guess missing `crio.sock` means its not running (or not installed but I think microshift pulls it as a dependency so we shouldn't end up in such situation)?
Pods might not be running, but should we attempt to remove image storage?

Reply:
That also uses crictl, which won't be able to talk to the backend if the socket is not there.

This case came up in the course of testing #2568, which uninstalls MicroShift and its dependencies.

Reply:
Another case is when microshift RPM is just installed, but not started. So, crio is not running at all, but the script can be run.